process/Style-guide.md

```diff
@@ -48,6 +47,26 @@ Other things to keep in mind when using Pylint:
 * Use textual pylint overrides e.g. "# pylint: disable=no-self-argument" instead of "# pylint: disable=E0213". For a list of overrides see: http://docs.pylint.org/features.html
 * The canonical pylint configuration for log2timeline project is generated by l2tdevtools. 
 
+### Type annotations
+
+* Use `import typing` not `from typing import ...`
```

Comment:
Note to self need to add `from __future__ import annotations` for Python 3.6 compatibility